### ISSUE TITLE
The alternating training problem

### ISSUE THREAD

**Issue by sibangde** [tryolabs/luminoth#191]
I can't find the alternating training step of faster rcnn, and actually my train result is very bad. So does this code has no alternating training step? If it has, can somebody indicate where it is? 
Thank you very much!

**Comment by joaqo** [comment]
If you wish to change the learning rate, or any other hyperparameter of a training job, just create your own custom config file, or modify one of the default ones. Eg.: FasterRCNN's config file: https://github.com/tryolabs/luminoth/blob/master/luminoth/models/fasterrcnn/base_config.yml
